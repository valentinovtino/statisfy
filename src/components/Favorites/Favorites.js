import React, { Component } from 'react';
import FoodHolder from '../foodHolder/foodHolder';
import { withRouter } from 'react-router';

const Favorites = () => {

  return (
    <div>
      {FoodHolder}
    </div>
  );
};

export default withRouter(Favorites);