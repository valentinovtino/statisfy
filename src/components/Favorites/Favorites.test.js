import Favorites from './Favorites';

describe('Favorites', () => {
  it('should render a card', () => {
    expect(Favorites).toMatchSnapshot();
  });
});