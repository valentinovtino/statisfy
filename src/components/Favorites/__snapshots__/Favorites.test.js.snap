// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Favorites should render a card 1`] = `[Function]`;
